docs/concepts/rate-limit-nft-concept.md

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_position: 4
+---
+
+# Rate Limiting
```

Comment:
should we place this section somewhere else? Maybe under 'SDK Reference' with 'Paying for Usage' or under 'Network' ?

Reply:
I am not sure, open to suggestions.

Reply:
let's move this to withib the SDK Reference section under 'Paying for Usage'